set.mm

```diff
@@ -68629,8 +68637,9 @@ pairs as classes (in set.mm, the Kuratowski encoding).  A more
 
   ${
     $d x A $.  $d x B $.  $d x F $.
-    $( Condition for the membership in the union of the range of a function.
-       (Contributed by Thierry Arnoux, 13-Nov-2016.) $)
+    $( Obsolete version of ~ elfvunirn as of 12-Jan-2025.  (Contributed by
+       Thierry Arnoux, 13-Nov-2016.)  (New usage is discouraged.)
+       (Proof modification is discouraged.) $)
     elunirn2 $p |- ( ( Fun F /\ B e. ( F ` A ) ) -> B e. U. ran F ) $=
```

Comment:
Please add a suffix OLD to the label, so that this teorem can be deleted after a yaer.